virt/virt-4-13-release-notes.adoc

```diff
@@ -200,6 +196,10 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-13-known-issues"]
 == Known issues
 
+//CNV-27681/BZ2183594 Migration of VMs does not complete with descheduler operator enabled
+// No BZ# because it is restricted to Red Hat.
+* The Descheduler is a Technical Preview and might cause failed migrations and unstable scheduling.
```

Comment:
Suggest linking to https://docs.openshift.com/container-platform/4.12/virt/virtual_machines/advanced_vm_management/virt-enabling-descheduler-evictions.html